src/app/revenue-share/revenue-share.component.html

```diff
@@ -1,34 +1,59 @@
+        <div class="col-md-12">
+            <div class="card">
+                <div class=" card-header">
+                    <div class=" card-title explainer-mini">
+                        Revenue share information
+                    </div>
+                </div>
+                <div class="card-body">
+                    <div class="float-left">
+                        <div class="explainer-mini">
+                            Project name
+                        </div>
+                        {{ projectName }}
+                    </div>
+                    <div class="float-right">
+                        <div class="explainer-mini">
+                            Amount invested
+                        </div>
+                        {{ amountInvested }}
+                    </div>
+                </div>
+            </div>
+            <button class="btn btn-danger w-100 py-3 mt-3" data-target="#modal-confirm-revenue" data-toggle="modal">
+                Verify and start revenue share
+            </button>
         </div>
-        <div class="card-body">
-          <div class="float-left">
-            <div class="explainer-mini">
-              Project name
+    </div>
+</div>
+
+
+<!-- Confirm Revenue Share Amount Modal -->
+<div #modalConfirmPopupDialog aria-hidden="true"
+     class="modal fade"
+     id="modal-confirm-revenue"
+     role="dialog"
+     tabindex="-1">
+    <div class="modal-dialog modal-dialog-centered" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h5 class="modal-title">Confirm revenue share amount</h5>
+                <button aria-label="Close" class="close" data-dismiss="modal" type="button">
+                    <span aria-hidden="true">&times;</span>
+                </button>
+            </div>
+            <div class="modal-body">
+                <input [(ngModel)]="amountInvestedConfirm" class="form-control px-4 py-4" id="revenue-confirm-amount" type="text">
             </div>
-            [Project name]
-          </div>
-          <div class="float-right">
-            <div class="explainer-mini">
-              Amount invested
+            <div class="modal-footer">
+                <button class="btn btn-secondary px-5 py-2" data-dismiss="modal" type="button">Cancel</button>
+                <button (click)="confirmButtonClicked()" class="btn btn-primary px-5 py-2" type="button">
+                    <i class="fas fa-check-circle mr-2"></i>
+                    Confirm
+                </button>
             </div>
-            [Amount invested]
-          </div>
         </div>
-      </div>
-      <div class="mt-3 explainer-mini">
-        Verify amount invested
-      </div>
-      <input class="mt-3 form-control w-100 py-4" type="text">
-      <button class="btn btn-danger w-100 py-3 mt-3">
-        Verify and start revenue share
-      </button>
     </div>
-  </div>
-</div>
\ No newline at end of file
+</div>
```

Comment:
There is a better and easier way of managing modals.

Create a new modal component (e.g. called RevenueShareConfirmModalComponent).
Set the template like:

```
<div class="modal-body">

</div>

<div class="modal-footer">

</div>
```

Inside the component, you can inject `BsModalRef` to interact with the modal component from inside.
```
import { BsModalRef, BsModalService } from "ngx-bootstrap/modal";
```

To open a modal from another component, you can use the following method:
```
import { BsModalService } from 'ngx-bootstrap/modal';

let modal = this.modalService.show(VideoAdModalComponent, {
      ignoreBackdropClick: true, // this is just example of additional configuration.
});
```

After that, you can make actions on the modal from its outer component.
This way, you can share modals (if needed) and isolate the code from the component.